test/e2e/SingleVoteGovernorE2E.t.sol

```diff
@@ -105,22 +105,15 @@ contract SingleVoteGovernorLifecycle is E2eTest {
 
         authorizer.renounceRole(ownerRole, address(this));
```

Comment:
Maybe explain which role the singlevote governor takes

Reply:
done :+1: